package.json

```diff
@@ -37,6 +37,8 @@
     "babel-eslint": "4.1.6",
     "babel-loader": "6.2.0",
     "babel-plugin-react-transform": "1.1.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.3.16",
```

Comment:
This one is unnecessary. It does only one trick - assigns `default` export to `module.exports`. Might be useful when your lib is used within commonjs environment (though it is better to make a simple commonjs wrapper for this).

We don't need it since we always use ES6 modules